ts/navigation/params/AppParamsList.ts

```diff
@@ -83,6 +83,7 @@ export type AppParamsList = {
   [ROUTES.WALLET_NAVIGATOR]: NavigatorScreenParams<WalletParamsList>;
   [SERVICES_ROUTES.SERVICES_NAVIGATOR]: NavigatorScreenParams<ServicesParamsList>;
   [ROUTES.PROFILE_NAVIGATOR]: NavigatorScreenParams<ProfileParamsList>;
+  [ROUTES.PROFILE_NEW_NAVIGATOR]: NavigatorScreenParams<ProfileNewParamsList>;
```

Comment:
As you can see at the bottom the params are taken directly into useIONavigation hook